corehq/apps/events/models.py

```diff
@@ -100,6 +103,12 @@ class Event(models.Model):
         choices=ATTENDEE_LIST_STATUS_CHOICES,
         default=NOT_STARTED,
     )
+    attendance_taker_ids = ArrayField(
```

Comment:
Are we sure we want a list of id's instead of a many-to-many relationship with CommCareUser? cc @kaapstorm